Bio/SeqUtils/CodonUsage.py

```diff
@@ -161,25 +164,32 @@ def _count_codons(self, fasta_file):
             self.codon_count = CodonsDict.copy()
 
             # iterate over sequence and count all the codons in the FastaFile.
-            for cur_record in SeqIO.parse(handle, "fasta"):
-                # make sure the sequence is lower case
-                if str(cur_record.seq).islower():
-                    dna_sequence = str(cur_record.seq).upper()
-                else:
-                    dna_sequence = str(cur_record.seq)
-                for i in range(0, len(dna_sequence), 3):
-                    codon = dna_sequence[i : i + 3]
-                    if codon in self.codon_count:
+            for record in SeqIO.parse(handle, "fasta"):
+                sequence = record.seq
```

Comment:
Does this need a ``.upper()`` to match the old behaviour?

(Noting as on another recent case, that the old code did not handle mixed case sequence properly)

Reply:
Indeed, it needs an `.upper()`; thanks!